src/main/java/spoon/support/reflect/code/CtLoopImpl.java

```diff
@@ -31,8 +32,9 @@ public CtStatement getBody() {
 		return body;
 	}
 
+	@SuppressWarnings("unchecked")
 	@Override
-	public <T extends CtLoop> T setBody(CtStatement body) {
+	public <T extends CtBodyHolder> T setBody(CtStatement body) {
 		if (body != null) {
```

Comment:
for two reasons:
1) because we want a 100% consistent behavior over all subinterfaces of CtBodyHolder
2) because it's 100% along the line of the new AlwaysInBlock contract introduced in #683